lib/rest-api.php

```diff
@@ -19,11 +19,10 @@
  * @return array Array of arguments for registering a post type.
  */
 function gutenberg_override_global_styles_endpoint( array $args ): array {
-	$args['rest_controller_class']           = 'WP_REST_Global_Styles_Controller_Gutenberg';
-	$args['revisions_rest_controller_class'] = 'Gutenberg_REST_Global_Styles_Revisions_Controller_6_6';
```

Comment:
@Mamaduka Am I doing this right? 😄 (Reference: https://github.com/WordPress/gutenberg/pull/65259)

Reply:
This looks good to me. 

The properties were added to support 6.5, but 6.6 specifies controllers in the post type registration now:

https://github.com/WordPress/wordpress-develop/blob/6.6/src/wp-includes/post.php#L488

Reply:
Looks correct to me as well 👍